include/internal/hwcaps.h

```diff
@@ -177,6 +177,9 @@ CPU_FEATURES_START_CPP_NAMESPACE
 #endif
 
 // https://elixir.bootlin.com/linux/latest/source/arch/riscv/include/uapi/asm/hwcap.h
+#define RISCV_HWCAP_32 0x32
+#define RISCV_HWCAP_64 0x64
+#define RISCV_HWCAP_128 0x128
```

Comment:
what is the purpose of these definitions and why represented in hex? Look like some magic numbers 
`0x128 -> 296` and I don't see that was declared by https://elixir.bootlin.com/linux/latest/source/arch/riscv/include/uapi/asm/hwcap.h

Reply:
Actually they are a placeholder to use introspection table. The hex number are invented. How can i define introspection table's line without hwcap?

Reply:
you can make the same as X86 impl
```C
LINE(X86_FPU, fpu, , , ) 
```